dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/inbound/HttpDispatcherHandler.java

```diff
@@ -114,7 +114,8 @@ public void exceptionCaught(ChannelHandlerContext context, Throwable cause) thro
             connection.stream(c.streamId()).close();
             return;
         } else if (cause instanceof IllegalArgumentException) {
-            FFDCFilter.processException(cause, HttpDispatcherHandler.class.getName() + ".exceptionCaught(ChannelHandlerContext, Throwable)", "1", context);
+            //Legacy doesnt throw ffdc on processNewInformation
```

Comment:
Legacy does not throw FFDC for malformed requests. Check build to verify no FATs need this FFDC. The transport.http_test FAT expects this FFDC to NOT be present, so it is being removed.